yojson.opam

```diff
@@ -1,22 +1,30 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "martin@mjambon.com"
 authors: ["Martin Jambon"]
-homepage: "https://mjambon.github.io/mjambon2016/yojson-doc/Yojson.html"
-bug-reports: "https://github.com/mjambon/yojson/issues"
-dev-repo: "https://github.com/mjambon/yojson.git"
-available: [ ocaml-version >= "4.02.3"]
+homepage: "https://github.com/ocaml-community/yojson"
+bug-reports: "https://github.com/ocaml-community/yojson/issues"
+dev-repo: "git+https://github.com/ocaml-community/yojson.git"
+doc: "https://ocaml-community.github.io/yojson/"
 build: [
-  ["jbuilder" "subst" "-n" name] {pinned}
-  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["dune" "subst" "-n" name] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
 ]
-
-build-test: [
-  ["jbuilder" "runtest" "-p" name]
-]
-
+run-test: ["dune" "runtest" "-p" name]
 depends: [
-  "jbuilder" {build}
+  "ocaml" {>= "4.02.3"}
+  "dune" {build}
   "cppo" {build}
   "easy-format"
   "biniou" {>= "1.2.0"}
 ]
+synopsis:
+  "Yojson is an optimized parsing and printing library for the JSON format"
+description: """
+It addresses a few shortcomings of json-wheel including 2x speedup,
```

Comment:
This is a rather historical description of yojson, which has outlasted the much-missed json-wheel library :-) Perhaps just describe yojson as a performant JSON serialisation library here?

Reply:
I think changing the description would indeed be good as some of us (myself included) might be too young to even know about `json-wheel`, but I think this can be done seperately. Also these changes actually come from #66!

Reply:
Yes, sorry about that. I've rebased on top of current master so it is less confusing now. I agree that updating description would be good, since I also don't know `json-wheel` 🙂